certified-connectors/OzyApprovals/apiDefinition.swagger.json

```diff
@@ -1 +1,1285 @@
+                },
+                "action_data": {
+                  "type": "object"
+                }
+              }
+            },
+            "headers": {
+              "Content-Type": {
+                "description": "Content-Type",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "summary": "Create a Ticket",
+        "description": "Create a Ticket within an approval stage.",
+        "operationId": "ActivateApprovalStage",
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "approval_process_request_id": {
+                  "type": "string",
+                  "description": "Request ID",
+                  "title": "Request ID",
+                  "x-ms-dynamic-values": {
+                    "operationId": "GetApprovalProcessRequests",
+                    "value-path": "id",
+                    "value-collection": "msg",
+                    "value-title": "name"
+                  },
+                  "format": "string"
+                },
+                "approval_stage_id": {
+                  "type": "string",
+                  "description": "Approval Stage",
+                  "title": "Approval Stage",
+                  "x-ms-dynamic-list": {
+                    "operationId": "GetProcessApprovalRequestStages",
+                    "itemValuePath": "id",
+                    "itemTitlePath": "name",
+                    "parameters": {
+                      "approval_process_request_id": {
+                        "parameterReference": "body/approval_process_request_id"
+                      }
+                    }
+                  },
+                  "x-ms-dynamic-values": {
+                    "operationId": "GetProcessApprovalRequestStages",
+                    "value-path": "id",
+                    "value-title": "name",
+                    "value-collection": "msg",
+                    "parameters": {
+                      "approval_process_request_id": {
+                        "parameter": "approval_process_request_id"
+                      }
+                    }
+                  }
+                },
+                "callback_url": {
+                  "type": "string",
+                  "description": "callback_url",
+                  "title": "",
+                  "x-ms-visibility": "internal",
+                  "default": "-"
+                }
+              },
+              "required": [
+                "approval_process_request_id",
+                "approval_stage_id",
+                "callback_url"
+              ]
+            }
+          }
+        ]
+      }
+    },
+    "/get_approval_process_requests": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
```

Comment:
Please use a proper description for the response. you can use something like "OK", "Sucess", "Bad request", etc.